install.sh

```diff
@@ -101,6 +101,28 @@ if [[ $? > 0 ]]
     fi
 fi
 
+#remove existing opendash desktop
```

Comment:
this might not be something that works for everyone (since not everyone might be running with a DE)

you would probs want to make this a configurable thing

Reply:
@rsjudka Moved it to a seperate file to install a desktop icon.